styles.css

```diff
@@ -0,0 +1,335 @@
+
+}
+
+.header-container {
+  background-image: url("images/header-image.jpg");
+  background-repeat: no-repeat;
+  display: flex;
+  background-color: grey;
+  width: 100%;
+  height: 30rem;
+  position: relative;
+}
+
+.header-logo {
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+  flex-wrap: wrap;
+  margin-left: 50px;
+  width: 20%;
+
+}
+
+.header-container p {
+  color: #81B741;
+  font-size: 1.2rem;
+  display: flex;
+  flex-wrap: wrap;
+  width: 100%;
+}
+
+
+
+.cart-container {
+  display: flex;
+
+}
+
+
+.cart-icon {
+  display: flex;
+  width: 2rem;
+  position: absolute;
+  top: 5%;
+  right: 10%;
+  color: black;
+
+
+}
+
+.checkout-button {
+  width: 2rem;
+  position: absolute;
+  top: 15%;
+  right: 15%;
+  padding: 1rem;
+  font-size: bold;
+}
+
+
+.nav {
+  background-color: #D3D3D3;
+  width: 100%;
+  height: 3rem;
+
+}
+
+  .nav ul li  {
+   list-style: none;
+   display: inline;
+   margin-left: 5%;
+   margin-top: 5%;
+   color: white;
+   padding: 2rem;
+   padding-top: 3rem;
+
+
+ }
+
+ li:after {
+   content: "/";
+   padding: 1rem;
+
+ }
+
+ .adopt-section-container {
+   display: flex;
+   flex-direction: row;
+   justify-content: space-between;
+   align-items: flex-start;
+   flex-wrap: wrap;
+   margin-left: 8%;
+   margin-right: 8%;
+   margin-top: 5%;
+   position: relative;
+
+ }
+
+
+ .adopt-section-image {
+   display: flex;
+   padding: 1rem;
+
+ }
+
+ .adopt-section-text {
+   display: flex;
+   padding: 1rem;
+   width: 60%;
+   flex-wrap: wrap;
+ }
+
+ .adopt-section-text h2 {
+   display: block;
+   width: 100%;
+   align-items: center;
+   justify-content: center;
+
+ }
+
+ .main-container {
+   position: relative;
+   display: flex;
+   flex-direction: row;
+   justify-content: space-evenly;
+   align-items: flex-start;
+   flex-wrap: wrap;
+   height: auto;
+   margin: 5% 8%;
+
+
+ }
+
+ .main-container h1 {
+   width: 100%;
+   padding: 2rem;
+
+ }
+
+ .dog-box {
+   display: flex;
+   align-items: flex-start;
+   flex-wrap: wrap;
+   width: 250px;
+   height: 400px;
+   min-width: 200px;
+   min-height: 200px;
+   margin: 10px 10px;
+   border: 1px solid green;
+
+ }
+
+ .dog-box-img {
+   display: flex;
+   flex-wrap: wrap;
+   width: 250px;
+
+ }
+
+ .dog-box-name {
+   display: flex;
+   flex-wrap: wrap;
+
+
+ }
+
+ .dog-box-cost {
+   display: flex;
+   flex-wrap: wrap;
+   color: gray;
+   display: flex;
+
+
+ }
+
+ .dog-box-cost {
+   display: flex;
+   flex-wrap: wrap;
+
+
+ }
+
+ .dog-box-info {
+   display: flex;
+   flex-wrap: wrap;
+
+
+ }
+
+ .dog-box.button {
+   display: flex;
+   flex-wrap: wrap;
+   width: 200px;
+   align-content: center;
+
+ }
+
+ .see-dog-button button {
+   width: 200px;
+ }
+
+
+ .blog-container {
+   display: flex;
+   flex-direction: row;
+   justify-content: flex-start;
+   align-items: flex-start;
+   flex-wrap: wrap;
+   margin-left: 8%;
+   margin-right: 8%;
+   margin-top: 5%;
+   position: relative;
+
+
+ }
+
+ .blog {
```

Comment:
the headline should be above the text, you may need to wrap those two elements in a container to have flexbox do it's magic right